src/Service/StorageCacheAbstractServiceFactory.php

```diff
@@ -14,7 +14,8 @@
  */
 class StorageCacheAbstractServiceFactory implements AbstractFactoryInterface
 {
-    public const CACHES_CONFIGURATION_KEY = 'caches';
+    public const CACHES_CONFIGURATION_KEY      = 'caches';
+    private const RESERVED_CONFIG_SERVICE_NAME = 'config';
```

Comment:
Why introduce constant? Lines 76 and 81 are not using it.

Reply:
because its way more readable than just having a random string somewhere.
The `get` does not need to check for the requested name as it shouldn't get passed due to the `canCreate` method returning false (implicit check).

I did a quick lookup with `phpdbg` and from what I can see, having this constant does not even produce more opcodes than using string here.

So IMHO, due to readability and same opcode amount, it should be fine having this around.